birthdays-project/BirthdayApp/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from django.http import Http404
+
+from .models import User
+
+
+def index(request):
+    userDB = User.objects.all()
+    return render(request, 'users/index.html', {
+        'user': userDB,
+    })
+
+
+def user_detail(request, id):
+    try:
+        userDB = User.objects.get(id=id)
```

Comment:
Again, the name. Better say `user` (or, if you refactor the model name, as suggested, `person`)